web/quickstart.html

```diff
@@ -415,7 +415,8 @@ <h2 id="notebook">Notebooks</h2>
     <p>You can also use Smile in your favorite Notebook.
         We recommend JupyterLab and provide <code>jupyterlab.sh</code>
         to setup the conda environment of Jupyter Lab for Smile with
-        kernels for Scala, Kotlin and Clojure. When you run
+        kernels for Scala, Kotlin and Clojure. The script is located at
+        <code>./shell/src/universal/bin/jupyterlab.sh</code>. When you run
```

Comment:
The script is in `bin` directory of release package. It is better not post the source code path here.